.gitlab/package_build/deb.yml

```diff
@@ -128,36 +106,13 @@ agent_deb-arm64-a6:
   before_script:
     - export RELEASE_VERSION=$RELEASE_VERSION_6
 
-agent_deb-arm64-a7:
-  extends: .agent_build_common_deb
-  rules:
-    - !reference [.on_a7]
-  stage: package_build
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_arm64$DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX:$DATADOG_AGENT_ARMBUILDIMAGES
-  tags: ["arch:arm64"]
-  needs:
-    [
-      "go_mod_tidy_check",
-      "build_system-probe-arm64",
-      "go_deps",
-      "generate_minimized_btfs_arm64",
-    ]
-  variables:
-    AGENT_MAJOR_VERSION: 7
-    PYTHON_RUNTIMES: "3"
-    PACKAGE_ARCH: arm64
-    DESTINATION_DEB: "datadog-agent_7_arm64.deb"
-    DESTINATION_DBG_DEB: "datadog-agent-dbg_7_arm64.deb"
-  before_script:
-    - export RELEASE_VERSION=$RELEASE_VERSION_7
-
 .iot_agent_build_common_deb:
   script:
     - source /root/.bashrc
     - !reference [.setup_ruby_mirror_linux]
     - !reference [.setup_python_mirror_linux]
     - !reference [.retrieve_linux_go_deps]
-    - echo "About to build for $RELEASE_VERSION_7"
+    - echo "About to build for $RELEASE_VERSION_6"
```

Comment:
Does it make sense to build the iot agent for agent 6? The IoT Agent 6 does not really exist since it does not embed Python

Reply:
Yes I agree with you, I initially wanted to dismiss all of this and decided I should make it in several parts, to have smaller PRs.